packages/core/core/src/utils/nuxt/index.ts

```diff
@@ -31,7 +33,18 @@ export const integrationPlugin = (pluginFn: NuxtPlugin) => (nuxtCtx: NuxtContext
       config.axios.baseURL = process.server ? ssrMiddlewareUrl || middlewareUrl : middlewareUrl;
     }
 
-    const client = axios.create(config.axios);
+    const keepAliveConfig = {
+      keepAlive: process.env.AXIOS_KEEP_ALIVE_ENABLED === 'true',
+      keepAliveMsecs: parseInt(process.env.AXIOS_KEEP_ALIVE_MSECS) || 1000,
+      maxFreeSockets: parseInt(process.env.AXIOS_MAX_FREE_SOCKETS) || 256
+    };
+
+    const client = axios.create({
+      ...config.axios,
+      httpAgent: new http.Agent(keepAliveConfig),
```

Comment:
I'm just thinking if there is a point to manage `httpAgent` and `httpsAgent` based on `process.env.NODE_ENV`. So if we are inside `DEVELOPMENT` we should care about `httpAgent` and in other case scenarios let's config `httpsAgent`. Wdyt?

Reply:
After consulting this with @filipsobol I get info that there is possibility to that production build is run in local environment, so it will be better to leave both.